### ISSUE TITLE
Creating a notification from a plugin

### ISSUE THREAD

**Issue by BobLuursema** [euvl/vue-notification#92]
I believe I am trying the same thing as @martiendt in #86, but it isn't working for me in my code below. There are no errors showing up but neither is a notification. I have tried replacing the calls to `Vue.prototype.$notify` with `Vue.notify` but it does not make any difference. I am noticing that in the Vue.js devtools console in Chrome there isn't an `add` event emitted, or at least not one noticed by the devtools.

@martiendt, if you got it to work, then I guess I must be calling something incorrectly.

```
let fetchPlugin = {}
fetchPlugin.install = function(Vue, options){
    Vue.prototype.$fetch = {

        get(url, processingFunction, show_loading=false, extra_options={}){
            if(show_loading){

                // This call does not work
                Vue.prototype.$notify({
                    group: 'loading',
                    title: 'Loading...',
                    type: 'success'
                })
            }

            let default_options = {
                credentials: 'same-origin',
            }
            options = Object.assign(default_options, extra_options)
            fetch(url, options)
                .then(function(response){
                    if(show_loading){

                        // This call works
                        Vue.prototype.$notify({
                            group: 'loading',
                            clean: true
                        })
                    }
                    if(!response.ok){ throw response }
                    return response.json()
                })
                .then(processingFunction)
                .catch(err => {
                    console.log(err)

                    // This call also works
                    Vue.prototype.$notify({
                        group: 'fetch',
                        title: 'An error occurred',
                        type: 'error'
                    })
                })    
        }
    }
}
export default fetchPlugin
```

**Comment by euvl** [comment]
Did you add notification container to your dom? Otherwise plugin does not know where you plonk your notifications

**Comment by BobLuursema** [comment]
Yes, that should be ok. Because the call does work when I put it inside the `.then` function...
Or the element maybe doesn't exists yet for some reason?

**Comment by euvl** [comment]
```
Or the element maybe doesn't exists yet for some reason?
```

This can be the case. You can test it by putting container as close to your root as possible and make for example a 1 sec delay before showing notification (after that you can narrow down what exactly is going on)

**Comment by BobLuursema** [comment]
Thanks @euvl, I managed to figure it out with your reply! I made it so that another call would put up an notification but only when I pressed the button, then the notification did work. So I looked at when I did the first call and that was in the `created` lifecycle hook of the root element. The notification element doesn't exist at that point in time of course. I changed that function to be called on the `mounted` lifecycle hook and then everything works great!
